app/controllers/admin/api/services/mapping_rules_controller.rb

```diff
@@ -10,11 +10,11 @@ class Admin::Api::Services::MappingRulesController < Admin::Api::Services::BaseC
 
   ##~ e = sapi.apis.add
   ##~ e.path = "/admin/api/services/{service_id}/proxy/mapping_rules.xml"
-  ##~ e.responseClass = "mapping_rule"
+  ##~ e.responseClass = "List[mapping_rule]"
   #
   ##~ op            = e.operations.add
   ##~ op.httpMethod = "GET"
-  ##~ op.summary    = "Mapping Rules List"
+  ##~ op.summary    = "Proxy Mapping Rules List"
```

Comment:
Do we say Proxy mapping rules List or Product or Service?

@thomasmaas

Reply:
We shouldn't use the 'Product' naming outside of APIAP, no?
About Proxy or Service, the description says "Returns the Mapping Rules of a Proxy" so it would be too weird to keep that description but the summary being "Service Mapping Rules List"